Backend/models.py

```diff
@@ -19,20 +21,363 @@ def setup_db(app, database_path=database_path):
+class CompanyWallet(db.Model):
+    __tablename__ = "wallet"
+    id = db.Column(db.Integer, primary_key=True)
+    balance = db.Column(db.Integer, nullable=False)
+    company = db.relationship('Company', backref='company_details', lazy=True)
+
+
+    def __init__(self, id, balance):
+        self.id = id
+        self.balance = balance
+
+    def insert(self):
+        db.session.add(self)
+        db.session.commit()
+
+    def update(self):
+        db.session.commit()
+
+    def delete(self):
+        db.session.delete(self)
+        db.session.commit()
+
+    def format(self):
+        return {
+            "id": self.id,
+            "balance": self.balance
+        }
+
+
+
+"""
+User Transactions
+
+"""
+
+
+class UserTransactions(db.Model):
+    __tablename__ = "users_transactions"
+    id = db.Column(db.Integer, primary_key=True)
+    # should we create a seperate column for type of transaction either debit or credit or just add it tp the description
+    description = db.Column(db.String, nullable=False)
+    # type = db.Column(db.String, nullable=False)
+    amount = db.Column(db.Integer, nullable=False)
+    # should we make it string or Boolean
+    status = db.Column(db.String, nullable=False)
+    # status = db.Column(db.Boolean, nullable=False)
+    date = db.Column(db.Date, nullable=False)
+    time = db.Column(db.Time, nullable=False)
+    user = db.relationship('User', backref='users_transactions', lazy=True)
+
+
+    def __init__(self, id, description, amount, status, date, time):
+        self.id = id
+        self.description = description
+        self.amount = amount
+        self.status = status
+        self.date = date
+        self.time = time
+
+    def insert(self):
+        db.session.add(self)
+        db.session.commit()
+
+    def update(self):
+        db.session.commit()
+
+    def delete(self):
+        db.session.delete(self)
+        db.session.commit()
+
+    def format(self):
+        return {
+            "id": self.id,
+            "description": self.description,
+            "amount": self.amount,
+            "status": self.status,
+            "date": self.date,
+            "time": self.time
+        }
+
+
+"""
+Company Transactions
 
 """
 
-class ModelName(db.Model):
-    __tablename__="model_name"
 
+class CompanyTransactions(db.Model):
+    __tablename__ = "company_transactions"
     id = db.Column(db.Integer, primary_key=True)
-    args= db.Column(db.String)
-    #other topics
+    # should we create a seperate column for type of transaction either debit or credit or just add it tp the description
+    description = db.Column(db.String, nullable=False)
+    # type = db.Column(db.String, nullable=False)
+    amount = db.Column(db.Integer, nullable=False)
+    # should we make it string or Boolean
+    status = db.Column(db.String, nullable=False)
+    # status = db.Column(db.Boolean, nullable=False)
+    date = db.Column(db.Date, nullable=False)
+    time = db.Column(db.Time, nullable=False)
+
```

Comment:
No, I believe we can incorporate that from inside the description as this will be both way i.e. "sent to Username" at the sender end and "received or sent from Username" at the receiver end

Reply:
Okay nice